.travis.yml

```diff
@@ -24,7 +30,4 @@ script:
   # Submit code coverage report to Coveralls
   - "mix coveralls.travis"
   # Run static code analysis
-  - "mix credo"
-  # Check code style
-  - "mix dogma"
-
+  - "mix credo"
```

Comment:
Please, delete the credo and dogma. We don't need them that much with the formatter and I believe they only make it harder to contribute.